app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -147,6 +149,9 @@ class HomeFragment : Fragment() {
 
         lifecycle.addObserver(sessionObserver)
 
+        if (finishOnboarding)
```

Comment:
nit: generally we always brace if/else statements, unless it's used as a one-line expression.

Reply:
Good call!